common/app/model/liveblog/BodyBlock.scala

```diff
@@ -116,6 +116,9 @@ case class BodyBlock(
   def publishedCreatedDate(timezone: DateTimeZone): Option[LiveBlogDate] = firstPublishedDate.orElse(createdDate).map(LiveBlogDate.apply(_, timezone))
 
   def publishedCreatedTimestamp(): Option[Long] = firstPublishedDate.orElse(createdDate).map(_.getMillis())
+
+  def referenceDateForDisplay(): Option[DateTime] = firstPublishedDate.orElse(publishedDate).orElse(createdDate)
```

Comment:
Relating this back to the conversation in Chat - is this an oversimplification? Do we need to concern ourselves with whether or not the `firstPublishedDate` is before (earlier than) `publishedDate`? I think the suggestion was:
> `if firstPublished is before article.publishDate then publishDate = articlePublishDate else firstPublished`
Sorry if I'm missing something!

Reply:
Discussed offline - I was indeed missing some context.